tests/test_stories.py

```diff
@@ -9,57 +9,141 @@
 import json
 
 poi1 = {
-    '_id' = 1
     'name': 'Himalayan Chimney',
-    'date': 'Date(2018, 2, 27)',
+    'date': '2018-02-02',
     'description': 'Yum',
     'map_year': 2018,
     'x_coord': 12,
     'y_coord': 43
 }
 
 poi2 = {
-    '_id' = 2
     'name': 'Mount Everest',
-    'date': 'Date(2018, 5, 15)',
+    'date': '2018-05-15',
     'description': 'Snow',
     'map_year': 2018,
     'x_coord': 15,
     'y_coord': 30
 }
 
 poi3 = {
-    '_id' = 3
     'name': 'Tampa Bay',
-    'date': 'Date(2018, 8, 3)',
+    'date': '2018-08-03',
     'description': 'Hot',
     'map_year': 2018,
     'x_coord': 50,
     'y_coord': 82
 }
 
-stories = [
-    {
-      '_id': 21,
-      'story_name': 'Angad Goes to Wisconsin',
-    },
-    {
-      '_id': 22,
-      'story_name': 'Amanda Takes her Medicine',
-    },
-    {
-      '_id': 23,
-      'story_name': 'Jeffy Goes to Fashion Show',
-    },
-    {
-      '_id': 24,
-      'story_name': 'Time for Hack4Impact',
-    }
-]
-            
-          
-    story = {
-    '_id': 25
+story_with_no_pois = {
     'story_name' : 'Jeffy Discovers the Dark side of the Moon'
 }
 
+story_with_pois = {
+    'story_name' : 'Angad Goes to Wisconsin',
+    'poi_ids' : []
+}
+
+story_edit = {
+      'story_name': 'Time for Hugs4Infants'
+}
+
+story_empty = {
+
+}
+
+
+os.environ['STORY_ID_1'] = '1' # this one doesn't have pois
+os.environ['STORY_ID_2'] = '2' # this one has the pois

```

Comment:
it'll be clearer elsewhere in the code if you directly named these `STORY_ID_WITHOUT_POIS` and `STORY_ID_WITH_POIS`